pkg/package/modfile.go

```diff
@@ -174,11 +176,36 @@ func (d *Dependency) FromKclPkg(pkg *KclPkg) {
 	d.LocalFullPath = pkg.HomePath
 }
 
+// The name of the local storage path is generated depending on the source of the package
+func (d *Dependency) GetPkgPathName() string {
```

Comment:
I think the function name is very strange.

Reply:
GetPkgPathName() has been removed.